### ISSUE TITLE
LCH colour sliders

### ISSUE THREAD

**Issue by notsidney** [sindresorhus/System-Color-Picker#20]
I love that your app treats LCH as a first-class citizen! I’ve been using LCH colours for different UI elements ever since I stumbled upon Lea Verou’s article, and I end up using her LCH colour picker https://css.land/lch/ a lot. It would be great to have this functionality in the sliders tab as well.

**Comment by sindresorhus** [comment]
Yes, this is planned. I'm not sure when I will get to this though.

**Comment by notsidney** [comment]
I just realized the LCH text field isn’t read-only, so I can write the specific LCH values I want, this is perfect for my use case! The LCH sliders are less needed for me now, but still a nice to have.

**Comment by yisibl** [comment]
Any news?

**Comment by sindresorhus** [comment]
@yisibl https://sindresorhus.com/blog/issue-bumping
